README.md

```diff
@@ -142,6 +142,56 @@ result = Hardmax(v)
 
 More examples can be found in the [docs/examples](docs/examples) directory.
 
+## ONNX Script Tools
```

Comment:
I think updating the very first paragraph would also be helpful. Saying something like "The repo covers ONNXScript + IR/Optimizer/Rewriter", expanded into a couple of sentences